trade-finance-manager-ui/server/middleware/validatePostAddPaymentRequestBody/index.ts

```diff
@@ -10,23 +10,47 @@ import {
 } from '../../helpers/premium-payments-table-checkbox-id-helper';
 import { PremiumPaymentsTableCheckboxId } from '../../types/premium-payments-table-checkbox-id';
 
+/**
+ * Checks if the given value is an object.
+ * @param body - The value to check.
+ * @returns True if the value is an object, false otherwise.
+ */
 const isRequestBodyAnObject = (body: unknown): body is object => !body || typeof body === 'object';
```

Comment:
this could be moved to a helper and used elsewhere as well and needs testing

Reply:
Wait - just spotted this. @Zainzzkk Didn't you already create a helper or use a lodash method for object checking? We should use the same thing here

Why do we need to check the request body is an object?

Reply:
Mine was to check if 2 objects are the same so this is different

Reply:
OK thanks.

Seems this PR is just adding documentation to this function so not a blocker

Reply:
Ah in that case can be left as is